packages/client/src/contexts/FiltersContext.tsx

```diff
@@ -0,0 +1,26 @@
+import { createContext, useCallback, useContext, useMemo, useState } from 'react';
+
+type SingleFilterType = {
+  name: string | null;
+  value: string | null;
+};
+
+const FiltersContextCurrent = createContext<{ filters: SingleFilterType[]; addFilters: (args: []) => void }>({
```

Comment:
Tutaj `[]` nie ma podanego typu, pewnie chodziło o:
```suggestion
const FiltersContextCurrent = createContext<{ filters: SingleFilterType[]; addFilters: (args: SingleFilterType[]) => void }>({
```